packages/@react-spectrum/link/src/Link.tsx

```diff
@@ -18,6 +18,10 @@ import styles from '@adobe/spectrum-css-temp/components/link/vars.css';
 import {useLink} from '@react-aria/link';
 import {useProviderProps} from '@react-spectrum/provider';
 
+/**
+ * Links allow users to navigate to a different location.
+ * They can be presented inline inside a paragraph or as standalone text.
```

Comment:
maybe `or as standalone content.`

Reply:
this is pulled from the spectrum website, so going to keep as is for consistency